client/src/components/Component.js

```diff
@@ -0,0 +1,27 @@
+export default class Component {
+  $dom;
+  state;
+  props;
```

Comment:
```suggestion
  _state;
  _props;
```

두개의 property 선언 앞에 언더바(_)를 붙여서 pretected의 효과를 주는 것이 좋을 것 같습니다.

Reply:
좋은 생각입니다. 그렇게 수정하겠습니다.